deploy/olm-catalog/service-telemetry-operator/manifests/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -122,63 +122,97 @@ spec:
                     description: Events related backend configuration.
                     properties:
                       elasticsearch:
-                        description: Events storage backend ElasticSearch
+                        description: Events storage backend Elasticsearch
                         properties:
                           certificates:
                             properties:
                               caCertDuration:
-                                description: The requested 'duration' (i.e. lifetime)
-                                  of the ElasticSearch CA Certificate. Minimum accepted
-                                  duration is 1 hour. Value must be in units accepted
-                                  by Go time.ParseDuration https://golang.org/pkg/time/#ParseDuration
+                                description: (DEPRECATED - Use ext* after STF 1.5.3)
```

Comment:
Just need to run `operator-sdk generate bundle --channel unstable --default-channel unstable` I believe to update the deploy/ directory with the updated contents from deploy/crds/